SETUP.md

```diff
@@ -1,4 +1,18 @@
-# Setting up python environment covid-data-model
+# Setting up dev environment for covid-data-model
+
+## Copy the source data
+
+Copy the source data from the `covid-data-public` repo to a sibling of your local `covid-data-model` directory. git-lfs must be
+installed to checkout a copy of `covid-data-public`.
+
+1. Install git-lfs
+  On mac, `brew install git-lfs`
+
+2. Clone the `covid-data-public` repo
+  ```
+  $ cd ..
+  $ git clone git@github.com:covid-projections/covid-data-public.git
+  $ cd -
```

Comment:
Maybe would be good to add a note that if you've cloned the repo before installing git lfs, you'll need to install git lfs, then do `git lfs pull` (pretty sure that's the command)

Reply:
Fixed with a319b4e